packages/graphql-transformers/src/openCrudWhereTransform/openCRUDtoSequelizeWhereTransformer.js

```diff
@@ -280,6 +258,26 @@ function correctManyRelationJoin(associationInfo, transformAssociationToNested,
 
 function transformWhereToNested(relatedEntityName, pathWithinSchema, associationAlias, relatedEntityFilter) {
   return transformWithSymbols.transformer({
+    transformers: [
+      {
+        cond: value => {
+          return value instanceof Sequelize.Utils.Fn;
```

Comment:
In handleRelation function there's a where clause for sub-query. Currently only Literal is used there, but on earlier versions of solution there also was Fn for EXISTS. When Sequelize primitives are processed by transformer, they are converted to plain Objects, and later are not recognized by Sequelize during SQL query generation, thus incorrect queries are generated. So here we need to preserve original types of Sequelize primitives so that generated SQL queries are correct. Right now there's no code having Fn used, do you want Fn transformer be removed? Transformer for Literal is needed anyway.